src/arrayMethodSort.js

```diff
@@ -4,8 +4,38 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
+  [].__proto__.sort2 = function (compareFunction) {
```

Comment:
just specify the default value
```suggestion
  [].__proto__.sort2 = function (compareFunction = (a, b) => String(a) > String(b) ? 1 : -1) {
```